### PR TITLE
PEP8 Fixes for Summarization.

### PR DESCRIPTION
**Pull request by souravsingh**

### CODE REVIEW

File: gensim/summarization/bm25.py [hunk 1]
```diff
@@ -18,7 +18,8 @@ class BM25(object):
 
     def __init__(self, corpus):
         self.corpus_size = len(corpus)
-        self.avgdl = sum(map(lambda x: float(len(x)), corpus)) / self.corpus_size
+        self.avgdl = sum(map(lambda x: float(len(x)), corpus)
```
Comment (tmylk) [inline gensim/summarization/bm25.py]: what is the reason for this change?

```
# Arguments on first line forbidden when not using vertical alignment.
foo = long_function_name(var_one, var_two,
    var_three, var_four)```

Comment (souravsingh) [inline gensim/summarization/bm25.py]: PEP8 error E501:Line longer than 79 characters.

Does this line need modification?

**Comment by piskvorky** [comment]
This PR has the same issues as the previous ones: makes code harder to read (unwanted line breaks, vertical indent).

File: gensim/summarization/bm25.py (hunk 1 above)
Comment (piskvorky) [inline gensim/summarization/bm25.py]: We ignore the 79 chars guideline. Put line breaks in places that are conceptually and visually meaningful. Try not to exceed 120 characters per line (unless necessary).

File: gensim/summarization/commons.py [hunk 2]
```diff
@@ -16,5 +16,6 @@ def build_graph(sequence):
 
 def remove_unreachable_nodes(graph):
     for node in graph.nodes():
-        if sum(graph.edge_weight((node, other)) for other in graph.neighbors(node)) == 0:
+        if sum(graph.edge_weight((node, other))
```
Comment (tmylk) [inline gensim/summarization/commons.py]: we don't check for lenght of line

File: gensim/summarization/keywords.py [hunk 3]
```diff
@@ -161,7 +161,8 @@ def _get_combined_keywords(_keywords, split_text):
         if word in _keywords:
             combined_word = [word]
             if i + 1 == len_text:
-                result.append(word)   # appends last word if keyword and doesn't iterate
```
Comment (tmylk) [inline gensim/summarization/keywords.py]: ditto

**Comment by tmylk** [comment]
do you know how to disable the 'line lenght' rule in a PEP8 check?

**Comment by souravsingh** [comment]
There is a command line argument called ``-max-line-length`` which can set the maximum line length for scripts(like 120 chars or so, default is 79).

**Comment by piskvorky** [comment]
There are syntax errors in this PR (`\ probability_matrix`, indent of diff's last line...), as well as unwanted changes (vertical indent). How was this tested?

I am really uneasy about all these "PEP8 fix" pull requests. It seems we're going in circles, introducing more problems than fixing.

File: gensim/summarization/bm25.py [hunk 4]
```diff
@@ -40,16 +40,16 @@ def initialize(self):
...
 
     def get_score(self, document, index, average_idf):
         score = 0
         for word in document:
             if word not in self.f[index]:
                 continue
             idf = self.idf[word] if self.idf[word] >= 0 else EPSILON * average_idf
-            score += (idf*self.f[index][word]*(PARAM_K1+1)
-                      / (self.f[index][word] + PARAM_K1*(1 - PARAM_B+PARAM_B*self.corpus_size / self.avgdl)))
+            score += (idf * self.f[index][word] * (PARAM_K1 + 1) /
```
Comment (piskvorky) [inline gensim/summarization/bm25.py]: No vertical indent.
Splitting the mega-expression into something saner (subexpressions) will help both readability and line length.

File: gensim/summarization/summarizer.py [hunk 5]
```diff
@@ -110,12 +110,17 @@ def _get_sentences_with_word_count(sentences, word_count):
     return selected_sentences
 
 
-def _extract_important_sentences(sentences, corpus, important_docs, word_count):
+def _extract_important_sentences(
```
Comment (piskvorky) [inline gensim/summarization/summarizer.py]: Definitely not.

File: gensim/summarization/summarizer.py [hunk 6]
```diff
@@ -110,12 +110,17 @@ def _get_sentences_with_word_count(sentences, word_count):
...
+        sentences,
+        corpus,
+        important_docs,
+        word_count):
     important_sentences = _get_important_sentences(sentences, corpus, important_docs)
 
     # If no "word_count" option is provided, the number of sentences is
     # reduced by the provided ratio. Else, the ratio is ignored.
-    return important_sentences if word_count is None else _get_sentences_with_word_count(important_sentences, word_count)
+    return important_sentences if word_count is None else _get_sentences_with_word_count(
```
Comment (piskvorky) [inline gensim/summarization/summarizer.py]: No.
